src/app/routes/utils/parseAvRoute/index.ts

```diff
@@ -10,7 +10,7 @@ const OPTIMO_ID_REGEX = /^c[a-zA-Z0-9]{10}o$/;
 const TIPO_ID_REGEX =
   /^(c[a-zA-Z0-9]{10,11}t)|([a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12})$/;
 
-const MEDIA_ID_REGEX = /^p[0-9a-z]{7,}/;
+const MEDIA_ID_REGEX = /^p[0-9a-z]{7}$/;
```

Comment:
Just thinking out loud here. This won't / shouldn't affect TV / Audio embed routes, right?

e.g. this is what we currently have:
TV: https://www.bbc.com/ws/av-embeds/media/hindi/bbc_hindi_tv/w172zm8b4tlm30d/hi
Live Radio: https://www.bbc.com/ws/av-embeds/media/bbc_persian_radio/liveradio/fa
Podcast: https://www.bbc.com/ws/av-embeds/media/portuguese/bbc_brasil/p0jdsmk2/pt-BR ❗ is this the scenario which this will resolve?

Now these URLs are currently out in the wild because amp, but I have no idea whether they will still be needed once we move over to the media loader (and because we have deamped these pages, and none of them should really support embeds)

I guess I am just wanting to sense check that this fix won't cause an issue with media on pages other than articles?

Reply:
https://www.bbc.com/ws/av-embeds/media/persian/bbc_persian_radio/p09pvvdc/fa should be OK, because `persian` won't match this regex (and never did I suppose since it was a min of 7)

Reply:
Good question, its been highlighted elsewhere that the IDs can be longer for WS specifically, so I'll try and think of a smarter approach to this.

Reply:
I guess the media ID can either be in the format `p1234567` or `w12345678901234`, right? So maybe just add that to the regex?

Reply:
I think these Tv/Audio routes will need extra work if we need to continue supporting them.

The BFF module only caters for hitting the /articles/ endpoint to fetch the data, which I imagine these Tv/Audio routes don't hit?

Reply:
No, they don't. So are you thinking, just worry about articles for now?

Reply:
Although these embed URLs will still need to work for the next couple of weeks until we're using the new MediaLoader

Reply:
I think it should be ok. The `getEmbedUrl` logic is in the MediaLoader (which could call into this parse function), but its only used to construct the `externalEmbedUrl` string for SMP. It won't be actually used for these Tv/Audio pages, so it should be fine.

The URLs will continue to work until Morph doesn't, so the timelines should line up for moving those page types away from Morph.

Reply:
Ah yes of course this makes sense now. We will not be implementing the external embed URL in our settings for the tv/audio/video pages. Thanks for clarifying 😌